templates/tfengine/components/resources/compute_instance_templates/main.tf

```diff
@@ -71,6 +71,16 @@ EOF
     {{end -}}
   }
   {{end -}}
+
+
+  {{if has . "metadata" -}}
```

Comment:
Could this be replaced with `hclField`?

Reply:
Map is currently not supported by `hclField`. https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/issues/666

Reply:
I just tried and you can still use `hcl`:
```
{{if has . "metadata" -}}
   metadata = {
      {{hcl .metadata}}
   }
{{end -}}
```

Reply:
Done